### ISSUE TITLE
Clear logs

### ISSUE THREAD

**Issue by anpic** [emanuele-f/PCAPdroid#375]
Please add a button or menu item to clear the connection log at any time

**Comment by emanuele-f** [comment]
Do you mean the list in the Connections tab?

**Comment by anpic** [comment]
> Do you mean the list in the Connections tab?

Yeap :)

**Comment by anpic** [comment]
Currently the list is cleared only after restarting. Which is not always possible. Especially if the PCAPdroid is added to Always-on VPN.

**Comment by anpic** [comment]
It would be even better to add the option to delete for each item in the list via the context menu or top panel. 
And even allow to select several as in the firewall blocklist and whitelist.

**Comment by emanuele-f** [comment]
I agree on adding the ability to reset the app state without restarting it. The other option to remove individual connections seems overcomplicate to me.

**Comment by anpic** [comment]
> The other option to remove individual connections seems overcomplicate to me.

You underestimate yourself ;)
Already your firewall is probably the best.
But without the possibility of flexible configuration, it's very difficult to use in complex scenarios.
It may be worth considering the possibility of customization through importing configs for example, in the JSON format. Without UI changes.

**Comment by emanuele-f** [comment]
>You underestimate yourself ;)

Ahaha, I mean it can be complicate also for the user to have this grade of configurability. For the time being, the simplicity of a single global list it's better from my point of view, both for code complexity and user experience

**Comment by emanuele-f** [comment]
>It may be worth considering the possibility of customization through importing configs for example, in the JSON format

Yes, but in part this is meant not to be simple. If people start using the app as an adblock, it could be removed from the store

**Comment by anpic** [comment]
> and user experience

If you start conducting surveys then you will learn a lot about the needs of users ;)





> it could be removed from the store

https://play.google.com/store/apps/details?id=app.greyshirts.firewall
https://play.google.com/store/apps/details?id=com.celzero.bravedns
and etc
Then it's time to remove them long ago ;)

**Comment by emanuele-f** [comment]
> https://play.google.com/store/apps/details?id=eu.faircode.netguard

At least for Netguard, I know for sure that you can't do that in the google play version https://github.com/M66B/NetGuard/blob/master/ADBLOCKING.md

**Comment by anpic** [comment]
> At least for Netguard, I know for sure that you can't do that in the google play version

It's just not necessary to loudly declare that you are adding to block ads ;)
Moreover, no one forbids adding in the form of plugins and configs. It all depends on the desire to realize or lack of it ;)

**Comment by ghost** [comment]
I second this. It would be a great feature. Maybe even have collapsible connections log that are time stamped e.g like every hour all the connections are collapsed into the single hour tab.

**Comment by Avrution** [comment]
Kind of surprised all these years later this still isn't implemented.

**Comment by emanuele-f** [comment]
@Avrution there are more important features that have been prioritized, this issue is not a priority among the open issues. This is an open source project, so everyone can contribute. If you can contribute in any way, either via coding or by supporting this request with more use cases, feel free to do it
